packages/select/src/vaadin-select.js

```diff
@@ -347,20 +351,7 @@ class SelectElement extends ElementMixin(
     super.ready();
 
     this._overlayElement = this.shadowRoot.querySelector('vaadin-select-overlay');
-    this._valueElement = this.shadowRoot.querySelector('[part="value"]');
-    this._toggleElement = this.shadowRoot.querySelector('[part="toggle-button"]');
-    this._nativeInput = this.focusElement.shadowRoot.querySelector('input');
-    this._nativeInput.setAttribute('aria-hidden', true);
-    this._nativeInput.setAttribute('tabindex', -1);
-    this._nativeInput.style.pointerEvents = 'none';
-
-    this.focusElement.addEventListener('click', (e) => {
-      const isHelperClick = Array.from(e.composedPath()).some((node) => {
-        return node.nodeType === Node.ELEMENT_NODE && node.getAttribute('slot') === 'helper';
-      });
-      this.opened = !this.readonly && !isHelperClick;
-    });
-    this.focusElement.addEventListener('keydown', (e) => this._onKeyDown(e));
+    this._toggleElement = this.shadowRoot.querySelector('[part~="input-field"]');
```

Comment:
`_toggleElement` sounds a bit unclear to me. There is a dedicated `[part=toggle-button]` part in the suffix slot, but that's not what `_toggleElement` is meant for.

That being so, I would suggest choosing a different naming such as, for example:

- `inputNode`
- `inputElement`
- `inputFieldNode`
- `inputFieldElement`

Reply:
Changed to `_inputContainer`.